packages/react-ui/src/atoms/Typography/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography as MuiTypography } from '@mui/material';
+
+const Typography = ({ italic = false, weight, children, ...otherProps }) => {
+  return (
+    <MuiTypography {...otherProps}>
+      <span style={{ fontWeight: weight, fontStyle: italic && 'italic' }}>
+        {children}
+      </span>
+    </MuiTypography>
+  );
+};
+
+Typography.propTypes = {
+  weight: PropTypes.oneOf([400 | 500 | 600]),
```

Comment:
I would propose to use same vocabulary as design i.e `regular`, `semiBold` ...

Reply:
Absolutely, I'll do it.